app/src/services/subscription.service.js

```diff
@@ -70,6 +70,8 @@ class SubscriptionsService {
             }
         }
 
+        body.params = { ...body.params, 'area': area.id }
```

Comment:
I don't think you need the single quotes around the attribute name.
```suggestion
        body.params = { ...body.params, area: area.id }
```

Reply:
Ah, eslint was failing silently in vscode.